app/javascript/packs/webauthn-unhide-signup.js

```diff
@@ -5,7 +5,7 @@ function unhideWebauthn() {
     const elem = document.querySelector('label[for=two_factor_options_form_selection_webauthn]');
     if (elem) {
       elem.hidden = false;
-      elem.classList.remove('hide');
+      elem.classList.remove('display-none');
```

Comment:
This is for 2FA options for signup, to present Security Key as an option only if the browser supports it.

Suggesting to keep this as display: none, since it's intended not to be available as an option by default.